firedrake/nullspace.py

```diff
@@ -83,6 +84,7 @@ def orthonormalize(self):
                 vec.array -= alpha
             vec.normalize()
         self.check_orthogonality()
+        self._ad_orthogonalized = True
```

Comment:
Should set this in orthogonalize too, otherwise you're really claiming orthonormal.

Reply:
Done

Reply:
Done